postman/src/core/clients/blockchain/IProvider.ts

```diff
@@ -1,16 +1,16 @@
 import { GasFees } from "./IGasProvider";
 
-export interface IChainQuerier<TransactionReceipt, Block, TransactionRequest, TransactionResponse, JsonRpcProvider> {
-  getCurrentNonce(accountAddress?: string): Promise<number>;
-  getCurrentBlockNumber(): Promise<number>;
+export interface IProvider<TransactionReceipt, Block, TransactionRequest, TransactionResponse, Provider> {
```

Comment:
@VGau as we discussed before, do u think we can get rid of these template class names `TransactionReceipt, Block, TransactionRequest, TransactionResponse, Provider` and use the ether.js class directly? This looks verbose, and as you mentioned, we indeed would always use ether.js and there's no need for supporting other libraries, wdyt? We could have another PR to follow that up if u think its easier